modules/dqt/ajax/saveQuery.php

```diff
@@ -69,7 +68,7 @@
                                     ': ' .
                                     $_REQUEST['QueryName'];
 }
-$fields = json_decode($_REQUEST['Fields']);
+$fields = json_decode(json_encode($_REQUEST['Fields']));
```

Comment:
PHP error here from using json_decode on an array would result in the frontend not getting a valid response and failing the save query for user.